spec/factories.rb

```diff
@@ -18,6 +18,14 @@
       further_information { %w[true false].sample }
       further_information_details { Faker::Lorem.paragraph_by_chars(number: 300) }
 
+      phone_number { '07700 900 982' }
+      address_line1 { '42' }
+      address_line2 { 'Much Wow Street' }
+      address_line3 { 'London' }
+      address_line4 { 'England' }
+      country { 'GB' }
+      postcode { 'SW1P 3BT' }
```

Comment:
Well done for using a valid postcode, which means this will pass later when we introduce postcode validation 📐